apps/rush-lib/src/rush-schema.json

```diff
@@ -70,6 +70,10 @@
       },
       "additionalProperties": false
     },
+    "telemetryEnabled": {
```

Comment:
It is alpha. Will update the wiki when it is seasoned. 

---
In reply to: [117849018](https://github.com/Microsoft/web-build-tools/pull/207#discussion_r117849018) [](ancestors = 117849018)